browser/ui/webui/new_tab_page/branded_wallpaper.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_NEW_TAB_PAGE_BRANDED_WALLPAPER_H_
+#define BRAVE_BROWSER_UI_WEBUI_NEW_TAB_PAGE_BRANDED_WALLPAPER_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+class BrandedWallpaperLogo {
+ public:
+  BrandedWallpaperLogo();
+  ~BrandedWallpaperLogo();
+
+  std::string imageUrl;
+  std::string altText;
+  std::string companyName;
+  std::string destinationUrl;
+};
+
+class BrandedWallpaper {
```

Comment:
I think we don't need additional data structure and using `NTPSponsoredImagesData` is sufficient.

Reply:
this is done now